layers/+lang/typescript/README.org

```diff
@@ -67,21 +76,22 @@ Currently tsserver doesn't pickup tsconfig.json file changes. You might need to
 
 ** Typescript Major Mode
 
-| Key Binding | Description                      |
-|-------------+----------------------------------|
-| ~SPC g b~   | jump back                        |
-| ~SPC g g~   | jump to entity's definition      |
-| ~SPC g t~   | jump to entity's type definition |
-| ~SPC g u~   | references                       |
-| ~SPC h h~   | documentation at point           |
-| ~SPC r r~   | rename symbol                    |
-| ~SPC s r~   | restart server                   |
+| Key Binding | Description                        |
+|-------------+------------------------------------|
+| ~SPC m =~     | Reformat the buffer              |
+| ~SPC m g b~   | jump back                        |
+| ~SPC m g g~   | jump to entity's definition      |
+| ~SPC m g t~   | jump to entity's type definition |
+| ~SPC m g u~   | references                       |
+| ~SPC m h h~   | documentation at point           |
+| ~SPC m r r~   | rename symbol                    |
+| ~SPC m s r~   | restart server                   |
 
 
 ** Reference Major Mode                          
 
-| Key Binding | Description                      |
-|-------------+----------------------------------|
-| ~C-j~       | find previous reference          |
-| ~C-k~       | find next reference              |
-| ~C-l~       | goto reference                   |
+| Key Binding | Description                        |
```

Comment:
I don't know if it is a good idea to manually align 2 different tables. Just press `tab`  and keep its size.